code/modules/projectiles/projectile/reusable.dm

```diff
@@ -36,24 +36,25 @@
 	icon_state = "foamdart_proj"
 	ammo_type = /obj/item/ammo_casing/caseless/foam_dart
 	range = 10
+	var/modified = 0
```

Comment:
to remember that the projectile and casing are modifier. So we don't have to use the ammo_casing var that I'm removing.